src/patternfly/components/Slider/slider.scss

```diff
@@ -0,0 +1,203 @@
+.pf-c-slider {
+  // rail
+  --pf-c-slider__rail--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-slider__rail--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-slider__rail-track--Height: #{pf-size-prem(4px)};
+  --pf-c-slider__rail-track--before--base--BackgroundColor: var(--pf-global--BorderColor--100);
+  --pf-c-slider__rail-track--before--fill--BackgroundColor: var(--pf-global--active-color--300);
+  --pf-c-slider__rail-track--before--BorderRadius: var(--pf-global--BorderRadius--lg);
+
+  // steps
+  --pf-c-slider__steps--FontSize: var(--pf-global--FontSize--sm);
+  --pf-c-slider__steps--Height: var(--pf-c-slider__steps--FontSize);
+
+  // step tick
+  --pf-c-slider__step-tick--Top: var(--pf-global--spacer--md);
+  --pf-c-slider__step-tick--Width: #{pf-size-prem(4px)};
+  --pf-c-slider__step-tick--Height: #{pf-size-prem(4px)};
+  --pf-c-slider__step-tick--BackgroundColor: var(--pf-global--BorderColor--200);
+  --pf-c-slider__step-tick--TranslateX: -50%;
+  --pf-c-slider__step--m-active__slider-tick--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-slider__step--first-child__step-tick--TranslateX: 0;
+  --pf-c-slider__step--last-child__step-tick--TranslateX: -100%;
+
+  // step label
+  --pf-c-slider__step-label--TranslateX: -50%;
+  --pf-c-slider__step-label--Top: calc(var(--pf-global--spacer--xl) + var(--pf-c-slider__rail-track--Height));
+  --pf-c-slider__step--first-child__step-label--TranslateX: 0;
+  --pf-c-slider__step--last-child__step-label--TranslateX: -100%;
+
+  // thumb
+  --pf-c-slider__thumb--Top: calc(var(--pf-c-slider__rail-track--Height) / 2 + var(--pf-global--spacer--md));
+  --pf-c-slider__thumb--Width: #{pf-size-prem(16px)};
+  --pf-c-slider__thumb--Height: #{pf-size-prem(16px)};
+  --pf-c-slider__thumb--Left: var(--pf-c-slider--value);
+  --pf-c-slider__thumb--BackgroundColor: var(--pf-global--primary-color--200);
+  --pf-c-slider__thumb--TranslateX: -50%;
+  --pf-c-slider__thumb--TranslateY: -50%;
+  --pf-c-slider__thumb--BorderRadius: var(--pf-global--BorderRadius--lg);
+  --pf-c-slider__thumb--BoxShadow--base:
+    0 0 0 2px #fff,
+    0 0 0 3px var(--pf-global--active-color--100);
+  --pf-c-slider__thumb--hover--BoxShadow: var(--pf-c-slider__thumb--BoxShadow--base);
+  --pf-c-slider__thumb--focus--BoxShadow: var(--pf-c-slider__thumb--BoxShadow--base);
+  --pf-c-slider__thumb--active--BoxShadow:
+    var(--pf-c-slider__thumb--BoxShadow--base),
+    0 0 2px 5px var(--pf-global--active-color--200);
+
+  // value
+  --pf-c-slider__value--MarginLeft: var(--pf-global--spacer--md);
+  --pf-c-slider__value--c-form-control--width-base: 3.5ch;
+  --pf-c-slider__value--c-form-control--width-chars: 2;
+  --pf-c-slider__value--c-form-control--Width: calc(var(--pf-c-slider__value--c-form-control--width-base) + (var(--pf-c-slider__value--c-form-control--width-chars) * 1ch));
```

Comment:
I think you need to add `+ (var(--pf-global--spacer--sm) * 2)` or the equivalent of `--pf-c-form-control--inset--base`. A 100% value doesn't fit for me

Reply:
![Screen Shot 2020-12-02 at 4 01 04 PM](https://user-images.githubusercontent.com/5385435/100931046-fb04fe00-34b7-11eb-8300-f280b4118a14.png)